stmhal/sdcard.c

```diff
@@ -229,7 +213,7 @@ mp_uint_t sdcard_read_blocks(uint8_t *dest, uint32_t block_num, uint32_t num_blo
             err = HAL_SD_CheckReadOperation(&sd_handle, 100000000);
         }
 
-        dma_deinit(sd_handle.hdmarx);
+        dma_deinit(&dma_SDIO_0_RX);
```

Comment:
The signature of dma_deint changed and requests now `dma_descr_t` -  the same that was used for setting up the dma handle.